cmd/storage-rest-client.go

```diff
@@ -156,6 +156,14 @@ func (client *storageRESTClient) Hostname() string {
 	return client.endpoint.Host
 }
 
+func (client *storageRESTClient) Endpoint() Endpoint {
+	return client.endpoint
+}
+
+func (client *storageRESTClient) Healing() bool {
```

Comment:
I don't understand exactly why this always returns false

Reply:
Because this functionality is to be used only for local disks @vadmeste